notificationworkerlambda/cdk/lib/senderworker.ts

```diff
@@ -241,8 +241,8 @@ export class SenderWorkerStack extends GuStack {
      * platform or app by talking to a different lambda handler function
      */
 
-    addWorker("ios", "ios", "com.gu.notifications.worker.IOSSender::handleChunkTokens")
-    addWorker("android", "android", "com.gu.notifications.worker.AndroidSender::handleChunkTokens")
```

Comment:
I did actually notice a bug here that I am fixing with this change, the editions params don't have the word `Live` in them so it is not a common suffix for all of them.